packages/luals-annotations/src/annotationBuilder.ts

```diff
@@ -0,0 +1,127 @@
+import luaLSType, { formatProperty } from "./luaLSType";
+import { Function, Property } from "@lest/docs";
+
+type FunctionRenderOptions = {
+	staticMethod?: boolean;
+	className?: string;
+};
+
+function renderParameterAnnotations(parameters: Property[]): string[] {
+	return parameters.map((param) => `---@param ${formatProperty(param, " ")}`);
+}
+
+function renderReturnAnnotations(returns: Property[]): string[] {
+	return returns.map((ret) => `---@return ${luaLSType(ret)} ${ret.name ?? ""}`);
+}
+
+function renderFunctionAliases(func: Function, staticMethod: boolean, className: string): string[] {
+	const { parameters = [] } = func;
```

Comment:
nit: Can you destructure this in the function signature? Also, can you replace `func.blah` below with destructuring for consistency's sake